src/components/AnonymousChat/child/ChatReplyView.tsx

```diff
@@ -54,7 +54,7 @@ const ChatReplyView = ({type, messageType, replyData}: ChatReplyViewProps) => {
       </View>
 
       <Text style={textStyle} numberOfLines={isReply ? 1 : undefined}>
-        {replyData?.text ?? replyData?.message}
+        {replyData?.attachments.length > 0 ? 'Photo' : replyData?.text ?? replyData?.message}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change introduces a hardcoded string 'Photo' when there are attachments in the reply data. This could lead to confusion if the attachment is not a photo (e.g., a document or video). Consider using a more generic term like 'Attachment' or dynamically determining the type of the attachment for display.
```diff
- {replyData?.attachments.length > 0 ? 'Photo' : replyData?.text ?? replyData?.message}
+ {replyData?.attachments.length > 0 ? 'Attachment' : replyData?.text ?? replyData?.message}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->